src/docfx/build/toc/TocMap.cs

```diff
@@ -159,21 +159,21 @@ private static (int subDirectoryCount, int parentDirectoryCount) GetRelativeDire
         private (FilePath[] tocs, Dictionary<FilePath, FilePath[]> docToTocs, List<FilePath> servicePages)
             BuildTocMap()
         {
-            using (Progress.Start("Loading TOC"))
+            using (var scope = Progress.Start("Loading TOC"))
```

Comment:
remove `{` and `}` if we are measuring the function?
```suggestion
            using var scope = Progress.Start("Loading TOC");
```